src/domains/idpay-app/08_k8s_configmaps.tf

```diff
@@ -89,6 +89,7 @@ resource "kubernetes_config_map" "rest-client" {
     idpay-reward-calculator-baseurl  = "http://idpay-reward-calculator-microservice-chart:8080"
     admissibility_ms_base_url        = "http://idpay-admissibility-assessor-microservice-chart:8080"
     idpay_merchant_host              = "http://idpay-merchant-microservice-chart:8080"
+    idpay-mock-baseurl               = "http://idpay-mock-microservice-chart:8080"
```

Comment:
To avoid variables with many different names, I would change to "base_url" since it seems to be the most frequent

Reply:
Sure, thanks